Make timeouts clearer for NPM DNS monitoring (#19919)
DataDog/documentation: content/en/network_monitoring/dns/_index.md

| **DNS requests**         | The number of DNS requests made from the client.                                                                         |
| **DNS requests / second** | The rate of DNS requests made by the client.                                                                             |
| **DNS response time**    | The average response time of the DNS server to a request from the client.                                                |

> | **Timeouts**             | The number of timed out DNS requests from the client (displayed as a percentage of all DNS responses). Note that these are not the same DNS timeouts reported by DNS clients or server, but a metric computed by NPM internally, and so may not align with DNS timeouts reported from outside of NPM.                |

Reviewer: ```suggestion
| **Timeouts**             | The number of timed out DNS requests from the client (displayed as a percentage of all DNS responses). <br  /><br />**Note**: These timeouts are a metric computed by NPM internally, and may not align with DNS timeouts reported from outside of NPM. They are not the same as the DNS timeouts reported by DNS clients or servers.                |
```
